pkg/ccl/backupccl/backupdest/backup_destination.go

```diff
@@ -143,21 +142,6 @@ func ResolveDest(
 		return ResolvedDestination{}, err
 	}
 
-	// At this point, the plannedBackupDefaultURI is the full path for the backup. For BACKUP
-	// INTO, this path includes the chosenSuffix. Once this function returns, the
-	// plannedBackupDefaultURI will be the full path for this backup in planning.
-	if len(incrementalFrom) != 0 {
-		// Legacy backup with deprecated BACKUP TO-syntax.
-		prevBackupURIs := incrementalFrom
-		return ResolvedDestination{
-			CollectionURI:    collectionURI,
-			DefaultURI:       plannedBackupDefaultURI,
-			ChosenSubdir:     chosenSuffix,
-			URIsByLocalityKV: urisByLocalityKV,
-			PrevBackupURIs:   prevBackupURIs,
-		}, nil
-	}
-
 	defaultStore, err := makeCloudStorage(ctx, plannedBackupDefaultURI, user)
 	if err != nil {
 		return ResolvedDestination{}, err
```

Comment:
as a follow up todo, you could clean up some of the error handling and comments below.